src/components/global/hamburguer.astro

```diff
@@ -0,0 +1,39 @@
+<div class="text-white">
+  <button id="open" class="w-auto h-auto lg:hidden">
+    <img
+      src="../../public/assets/img/hamburguer-menu.svg"
+      alt="Hamburguer Menu"
```

Comment:
Typo in alt text, this file's name, and the image's filename: "hamburger"

Reply:
fixed !